docs/documentations/test-runner.md

```diff
@@ -0,0 +1,7 @@
+Para la elección del test runner, teniendo en cuenta que los tests tienen que ser ejecutados una gran cantidad de veces durante el desarrollo del proyecto, el criterio de elección y de búsqueda se centra en el rendimiento. También se tiene en cuenta la popularidad y documentación del test runner.
+
+- **Jasmine**: es un testing framework de JavaScript con test runner integrado que  funciona sin necesidad de una librería de aserción adicional proprocionando un sintaxis limpio y fácil de entender. Es rápido para los tests síncronos pero el rendimiento baja para los tests asíncronos.
```

Comment:
¿Por qué no ponéis enlaces a los sistemas que mencionáis?

Reply:
¿Tus tests son asíncronos o asíncronos? Si no lo sabes, ¿por qué mencionas esto?